module/VuFind/src/VuFind/ILS/Driver/Folio.php

```diff
@@ -1463,16 +1463,17 @@ protected function getInstructorIds($courseListingId)
     public function findReserves($course, $inst, $dept)
     {
         $retVal = [];
+        $idType = $this->getBibIdType();
 
         // Results can be paginated, so let's loop until we've gotten everything:
         foreach ($this->getPagedResults(
             'reserves',
             '/coursereserves/reserves'
         ) as $item) {
-            try {
-                $bibId = $this->getBibId(null, null, $item->itemId);
-            } catch (\Exception $e) {
-                $bibId = null;
+            if ($idType == 'hrid') {
```

Comment:
If you wanted to make this even more concise, you could consider something like:

```
$idProperty = $this->getBibIdType() === 'hrid' ? 'instanceHrid' : 'instanceId';
$bibId = $item->copiedItem->$idProperty ?? null;
```

Reply:
Works for me, please edit as you like.

Reply:
Thanks! It looks like I don't have write access to this branch, so I can't make changes directly. If you'd like me to, then please give me access. If not, I can just modify as part of the merge-to-dev process. Either way, I'm currently waiting to get some course data set up in my local test environment, and I'll give this a closer look once that's ready.